packages/core/EventManager.ts

```diff
@@ -185,9 +187,9 @@ export default class EventManager {
             meetingPassword: true,
             meetingUrl: true,
             externalCalendarId: true,
+            credentialId: true,
           },
         },
-        destinationCalendar: true,
```

Comment:
Since we have the credential ID from the booking reference we don't need to grab the destination calendar

Reply:
Why not? is the destination calendar related to the credential? What happen to team event types where you can override the destinationCalendar per event-type?

Reply:
Since we're rescheduling a booking that means that the booking references are already created which means it should contain the specific credential id for the destination calendar of that event type.